getting error on build:  "bacio::bacio_4" is imported but not globally visible.

```
cmake .. -DCMAKE_PREFIX_PATH="/usr/local/g2c;/usr/local/NCEPLIBS-bacio-2.4.1;/usr/local/jasper-2.0.32"
-- Found bacio: /usr/local/NCEPLIBS-bacio-2.4.1/lib/libbacio_4.a (found version "2.4.1")
CMake Error at CMakeLists.txt:30 (add_library):
  add_library cannot create ALIAS target "bacio::bacio" because target
  "bacio::bacio_4" is imported but not globally visible.


-- Configuring incomplete, errors occurred!
See also "/home/ed/NCEPLIBS-g2/b/CMakeFiles/CMakeOutput.log".
```

@kgerheiser have you seen this before?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 # This is the main CMake file for NCEPLIBS-g2.
 #
 # Mark Potts, Kyle Gerheiser
-cmake_minimum_required(VERSION 3.15)
+cmake_minimum_required(VERSION 3.18)
 
 file(STRINGS "VERSION" pVersion)
 

